generate.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+use Nette\PhpGenerator\ClassType;
```

Comment:
A namespace is very much required. Maybe consider using an anonymous class to avoid any symbol detection (by downstream projects) overall?

Reply:
Marco, please don't look at generate, it's not even pre-beta, I'm here just to POC the results of the generation yet :-)

Reply:
That's fine, just commenting on it 😛